ocs_ci/utility/aws.py

```diff
@@ -1234,3 +1301,24 @@ def delete_cluster_buckets(cluster_name):
                 logger.error(e)
         else:
             logger.info("No matches found for pattern %s", pattern)
+
+
+def get_stack_name_from_instance_dict(instance_dict):
+    """
+    Get the stack name by the given instance dictionary from AWS
+
+    Args:
+        instance_dict (dict): The instance dictionary from AWS
+
+    Returns:
+        str: The stack name of the given instance dictionary from AWS.
+             If not found returns None
+    """
+    tags = instance_dict.get('Tags', [])
+    stack_name = None
+
+    for tag in tags:
+        if tag.get('Key') == 'aws:cloudformation:stack-name':
```

Comment:
`'aws:cloudformation:stack-name'` is already saved as a constant in ocs_ci/cleanup/aws/defaults.py:16

I suggest moving this constant to ocs_ci/ocs/constants.py and using it here